lib/registry.js

```diff
@@ -118,7 +119,17 @@ util.inherits(RegistryItem, Object);
 Object.freeze(RegistryItem);
 Object.freeze(RegistryItem.prototype);
 
-
+/**
+ * Get the path to system's reg.exe. Useful when another reg.exe is added to the PATH
+ * Implemented only for Windows
+ */
+function getRegExePath() {
+    if (process.platform === 'win32') {
+        return path.join(process.env.windir, 'system32', 'reg.exe');
```

Comment:
I am fine with this solution. I still only want to check if the arch option works properly. I guess I have some time this weekend.